scylla/src/transport/connection.rs

```diff
@@ -1628,6 +1628,7 @@ mod tests {
     ///    Then use query_iter with page_size set to 7 to select all 100 rows.
     /// 3. INSERT query_iter should work and not return any rows.
     #[tokio::test]
+    #[cfg_attr(scylla_cloud_tests, ignore)]
```

Comment:
Ah, I suggested using `cfg_attr` not having read beforehand what it does precisely... While what you did here will work, I think it will be slightly clearer if you use `#[cfg(not(scylla_cloud_tests))]`.

Reply:
Ok, changed it to `cfg(not(...))`.